src/hott/06-contractible.rzk.md

```diff
@@ -314,17 +316,17 @@ For example, we prove that based path spaces are contractible.
 
 -- The center of contraction in the based path space is `(a , refl)`
 #def based-paths-center
-  (A : U)         -- The ambient type.
-  (a : A)         -- The basepoint.
-  : Σ (x : A) , a = x
-  := (a , refl)
+  ( A : U)         -- The ambient type.
+  ( a : A)         -- The basepoint.
+  : Σ ( x : A) , a = x
+  := ( a , refl)
 
 -- The contracting homotopy.
 #def based-paths-contracting-homotopy
-  (A : U)                     -- The ambient type.
-  (a : A)                     -- The basepoint.
-  (p : Σ (x : A) , a = x)      -- Another based path.
-  : (based-paths-center A a) =_{Σ (x : A) , a = x} p
+  ( A : U)                     -- The ambient type.
+  ( a : A)                     -- The basepoint.
+  ( p : Σ ( x : A) , a = x)      -- Another based path.
+  : ( based-paths-center A a) =_{Σ ( x : A) , a = x} p
```

Comment:
`(based-paths-center A a) =_{Σ (x : A) , (a = x)} (p)`

Reply:
I'm a bit unsure how to handle `=_{...}` currently. What I've been doing so far is to mostly introduce a line break before and after. I.e.
```rzk
...
=_{...}
...
```
But I'm open to other suggestions.

Reply:
(I'm not suggesting that you should do it here)

Reply:
In a sense this seems to be a relic of an older state of rzk when we occasionally needed to supply the name of the type to get things to typecheck @fizruk . I haven't found this to be an issue recently and when updating the style ended up deleting some of these type annotations without trouble.